### PR TITLE
Dynamic image import

### PR DESCRIPTION
**Pull request by agdust**
I've grouped all images and uploaded some new quality ones

(I've rebased it on #49 to avoid conflicts and fixes, so it must be merged after it)

### CODE REVIEW

File: app/util/image.js [hunk 1]
```diff
@@ -1,14 +1,52 @@
...
+  const path = character ? 'front.' + character : 'back';
+
+  return `/image/card/vagabond-character/card-vagabond_character-${path}.jpg`;
+}
+
+export function getFactionBoardPath(faction, back) {
+  const side = back ? 'back' : 'front';
+
+  return `/image/board/board-${faction}-${side}.jpg`;
+}
```
Comment (foxfriends) [inline app/util/image.js:52]: I would prefer this to be two functions, one for front and one for back. The meaning of the final boolean is unclear when called.

**Review by foxfriends — CHANGES_REQUESTED** [review]
Just this one small thing. 

It's too bad we cannot use a function for the CSS image references, maybe those can go in a file of CSS variables though? That can be for next time if ever though.

**Comment by agdust** [comment]
> we cannot use a function for the CSS image references

We still can define custom-properties in component itself like `style={--image-we-need: url(${getImagePath()})}`

I think this way covers any problem with css images, because there will be few of them (most of images will be defined in the style attribute). And if we'll try to pass _all_ images to css, we'll get this huge files that we've got rid just now

**Comment by foxfriends** [comment]
> > we cannot use a function for the CSS image references
> 
> We still can define custom-properties in component itself like `style={--image-we-need: url(${getImagePath()})}`
> 
> I think this way covers any problem with css images, because there will be few of them (most of images will be defined in the style attribute). And if we'll try to pass _all_ images to css, we'll get this huge files that we've got rid just now

Let's just leave this alone for now, there's not too many of them. I think that could be done just using `background-image` in the style property most of the time, not needing the custom one.

**Review by foxfriends — APPROVED** [review]

**Comment by foxfriends** [comment]
Thanks!

**Pull request merged**